.vscode/settings.json

```diff
@@ -25,5 +25,6 @@
       }
     },
   ],
-  "java.test.defaultConfig": "WPIlibUnitTests"
+  "java.test.defaultConfig": "WPIlibUnitTests",
+  "java.jdt.ls.vmargs": "-XX:+UnlockExperimentalVMOptions"
```

Comment:
If this flag was added automatically by the upgrade to 2022.2.1, then it's all good. If we added this then we should probably think twice.

Reply:
I had to add this myself to get the language server to start, otherwise I had no language support whatsoever. The new update added a VM argument, and I suspect WPILIB forgot to install the additional argument when upgrading the project.